src/backend/opencl/unwrap.cpp

```diff
@@ -51,4 +52,48 @@ INSTANTIATE(uchar)
 INSTANTIATE(char)
 INSTANTIATE(short)
 INSTANTIATE(ushort)
+#undef INSTANTIATE
+
+template <typename T>
+Array<T> unwrap_dilated(const Array<T> &in, const dim_t wx, const dim_t wy,
```

Comment:
I believe we follow camelCase convention for backend API or all normal C++ functions. Of course there are some exceptions from the past, but I think we should stick to the convention.